.github/workflows/tr.unit.tests.yaml

```diff
@@ -56,8 +56,8 @@ jobs:
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
             ${{ runner.os }}-maven-
-      - name: Run unit tests
-        uses: ./.github/actions/tr-tests
+      - name: Run Traffic Router unit tests
```

Comment:
Since this GitHub Actions step rename is not related to updating Grafana dashboard links, would you mind opening a separate PR for this?